### ISSUE TITLE
Dropdown with selectedKeys prop causing aria-activedescendant issue

### ISSUE THREAD

**Issue by levymsft** [microsoft/fluentui#15383]
### Environment Information

- **Package version(s)**: 7.144.2
- **Browser and OS versions**: Edge Version 85.0.564.68 (Official build) (64-bit) / Windows 10

### Describe the issue:
[Microsoft Accessibility Insights for Web plugin](https://microsoftedge.microsoft.com/addons/detail/accessibility-insights-fo/ghbhpcookfemncgoinjblecnilppimih) is reporting issues when we have a Dropdown  component with pre-selected items opened. 

[Codepen ](https://codepen.io/levymoreira/pen/vYKBMBP?editors=1000)
```
export const App: React.FunctionComponent = () => {
  const optionsList: IDropdownOption[] = [
    { key: 1, text: 'One' },
    { key: 2, text: 'Two' }
  ];

  return (
    <div>
      <label id='testDropdownLabel'>Test dropdown</label>
      <Dropdown
          id='testDropdown'
          aria-labelledby='testDropdownLabel'
          ariaLabel='Test'
          selectedKeys={[1]} // <- Just happens with selectedKeys
          options={optionsList}
          multiSelect={true}
        >
      </Dropdown>
    </div>
  );
};
```

With selectedKeys
![image1](https://user-images.githubusercontent.com/72461213/95233789-2db6b100-07fd-11eb-9ef2-7040005c071d.png)

Without selectedKeys
![image2](https://user-images.githubusercontent.com/72461213/95233861-327b6500-07fd-11eb-9ad9-69b58941646f.png)

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/levymoreira/pen/vYKBMBP?editors=1000
Just click on the Dropdown to list the options and inspect the element, 
a property `aria-activedescendant="testDropdown-list0"` will be added to the dropdown div causing the issue.

#### Actual behavior:
Accessibility validations failing for a simple Dropdown without any adicional elements.
#### Expected behavior:
Accessibility passing. 

### Documentation describing expected behavior
1. Install the accessibility plugin on edge/chrome https://microsoftedge.microsoft.com/addons/detail/accessibility-insights-fo/ghbhpcookfemncgoinjblecnilppimih
2. Run the snipped of code above using the latest version 
3. Select the dropdown to open the options
4. Click on the plugin icon and select `FastPass`

**Comment by paulgildea** [comment]
@levymsft Thanks for the issue!

@smhigley FYI

**Comment by smhigley** [comment]
Seems like this should be a combobox and not a button (not only because of this one issue)
